partitura/io/exportaudio.py

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 """
-This module contains methods to synthesize Partitura object to wav using
-additive synthesis
+This module contains methods to synthesize Partitura object to wav.
```

Comment:
either:
-  This module contains methods to synthesize a Partitura object to wav.
-  This module contains methods to synthesize Partitura objects to wav.